VimeoUpload/Upload/Model/VideoSettings.swift

```diff
@@ -29,18 +29,34 @@ import Foundation
 class VideoSettings: NSObject
 {
     var title: String?
+    {
+        didSet
+        {
+            self.title = trim(self.title)
```

Comment:
@cjlarsen A couple comments to address, 

Whenever we invoke an instance method we prefix it with `self.` (we should add that here)